Plugin/Model/PagePlugin.php

```diff
@@ -3,6 +3,7 @@
 namespace Mooore\WordpressIntegrationCms\Plugin\Model;
 
```

Comment:
In deze change kan ik het niet zien, maar in de file viewer wel: Hier wordt geen strict_typing toegepast. Graag toepassen.

Reply:
Done :+1: